cocos/platform/win32/CCApplication-win32.cpp

```diff
@@ -83,19 +95,32 @@ int Application::run()
     // Retain glview to avoid glview being released in the while loop
     glview->retain();
 
+    LONGLONG interval = 0LL;
+    LONG waitMS = 0L;
+
+    LARGE_INTEGER freq;
+    QueryPerformanceFrequency(&freq);
+
     while(!glview->windowShouldClose())
     {
         QueryPerformanceCounter(&nNow);
-        if (nNow.QuadPart - nLast.QuadPart > _animationInterval.QuadPart)
+        interval = nNow.QuadPart - nLast.QuadPart;
+        if (interval >= _animationInterval.QuadPart)
         {
-            nLast.QuadPart = nNow.QuadPart - (nNow.QuadPart % _animationInterval.QuadPart);
-            
+            nLast.QuadPart = nNow.QuadPart;
```

Comment:
I don't know why should subtract `nNow.QuadPart % _animationInterval.QuadPart` here.
I think the correct logic should be `nLast.QuadPart = nNow.QuadPart`.